LICENSE

```diff
@@ -19,3 +19,33 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+---
+
+The `server` crate is a fork of `ruff_server`:
+
+- URL: https://github.com/astral-sh/ruff/tree/main/crates/ruff_server
+- Commit: aa429b413f8a7de9eeee14f8e54fdcb3b199b7b7
+- License: MIT
+
+MIT License
+
+Copyright (c) 2022 Charles Marsh
+
```

Comment:
I tried the idea of putting some kind of header info in each file but it was not maintainable with the amount of changes I was also making to those files (deleting some, merging others, completely rewriting some, etc)